### PR TITLE
Add list all students with specified tags feature

### PR DESCRIPTION
**Pull request by ElijahS67**
Displaying persons in alphabetical order (sorting) is not possible (?) due to `filteredPersons` in `ModelManager.java` being `final`.
So currently it only lists all persons that contain the specified tags.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.PersonContainsTagPredicate;
+
+/**
+ * Lists persons that match all tags specified by the user.
```
Comment (carriezhengjr) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:9]: "persons" could be changed to "classmates", "user" to "student" 🤔

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists persons that match all tags specified\n"
```
Comment (carriezhengjr) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:14]: "persons" could be changed to "classmates" to be more specific to Mass Linkers. 🤔

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            + "Example: " + COMMAND_WORD + " team1";
+
+    public static final String MESSAGE_SUCCESS = "Listed all person(s) with the specified tag(s)";
```
Comment (carriezhengjr) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:17]: "person(s)" could be changed to "classmate(s)".  🤔

File: src/main/java/seedu/address/model/person/PersonContainsTagPredicate.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+/**
+ * Tests that a {@code Person}'s {@code Tag} matches all the tags given.
+ */
+public class PersonContainsTagPredicate implements Predicate<Person> {
```
Comment (carriezhengjr) [inline src/main/java/seedu/address/model/person/PersonContainsTagPredicate.java:11]: Perhaps person used in this class could also be changed to classmate to suit Mass Linkers? 🤔

**Review by carriezhengjr — COMMENTED** [review]
Works well, tested it on my own laptop. 👍 
Just very small nitpicks on words like person, which could be changed to classmate to cater to Mass Linkers.

File: src/main/java/seedu/address/logic/commands/SortCommand.java (hunk 1 above)
Comment (ElijahS67) [reply src/main/java/seedu/address/logic/commands/SortCommand.java:9]: Ok, will change and push soon!

**Pull request closed without merging**